src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolFinalStep.tsx

```diff
@@ -55,32 +47,29 @@ const TableToolFinalStep = ({
   );
   const publication = fullPublication?.publication;
 
-  const handlePermalinkClick = async () => {
-    if (!currentTableHeaders) {
-      return;
+  const getMethodologyLinks = () => {
```

Comment:
The second problem I'm having looks to be a bug in this method causing `TableToolFinalStep.test.tsx` to fail test `renders the external methodology link correctly`. The external methodology link doesn't appear to be rendered.